packages/protocol-http/package.json

```diff
@@ -54,5 +55,8 @@
   },
   "typedoc": {
     "entryPoint": "src/index.ts"
+  },
+  "publishConfig": {
+    "directory": "_release/package"
```

Comment:
nit: Should a private directory be `.release/` rather than `_`?

Reply:
Changed to .release in [fe6a59d](https://github.com/awslabs/smithy-typescript/pull/749/commits/fe6a59dfbe9286b7d38f3df5461921d70ac01ddd)